src/main/resources/application.yml

```diff
@@ -0,0 +1,17 @@
+server:
+  port: 8082
+
+crime.portal:
+  base.url: ${CRIME_PORTAL_BASE_URL:http://localhost:8081}
```

Comment:
unnecessary env variable reference since env variables automatically override application properties

Reply:
Cool.  I was copying the properties setup from offender-case-notes: https://github.com/ministryofjustice/offender-case-notes/blob/master/src/main/resources/application-dev.yml

Reply:
:man_facepalming: not spotted that before in case notes service, will fix :+1: